### PR TITLE
TRD reduce spectrum count

### PR DESCRIPTION
**Pull request by bazinski**
Collapse the 540 cluster spectra, 1 for each chamber, into a singular 2d histogram.
This should lighten the load, we still end up with the same information.

Additonally add the currently used json file from my home directory to this github repo. Its sampling rate must be reduced still.

### CODE REVIEW

File: Modules/TRD/TRDQC.json [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        "type": "2",
+        "periodName": "",
+        "passName": "",
+        "provenance": "qc"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
```
Comment (Barthelemy) [inline Modules/TRD/TRDQC.json:22]: The url for consul should be left empty. 
```suggestion
        "url": ""
```

**Review by Barthelemy — COMMENTED** [review]
Thank you Sean. Just one little change required.

File: Modules/TRD/TRDQC.json (hunk 1 above)
Comment (bazinski) [reply Modules/TRD/TRDQC.json:22]: Ah shot, was never what was supposed to go in that stuff in anycase ;-)

**Comment by Barthelemy** [comment]
The failing test is unrelated, merging.

**Pull request merged**